cmake/ConkyPlatformChecks.cmake

```diff
@@ -249,14 +253,39 @@ endif(BUILD_HTTP)
 
 if(BUILD_NCURSES)
   set(CURSES_NEED_NCURSES TRUE)
-  include(FindCurses)
 
-  if(NOT CURSES_FOUND)
-    message(FATAL_ERROR "Unable to find ncurses library")
-  endif(NOT CURSES_FOUND)
+  find_path(NCURSES_INCLUDE_PATH
+    NAMES ncurses.h curses.h
+    PATH_SUFFIXES ncurses
+    PATHS /usr/include /usr/local/include /usr/pkg/include
+  )
 
-  set(conky_libs ${conky_libs} ${CURSES_LIBRARIES})
-  set(conky_includes ${conky_includes} ${CURSES_INCLUDE_DIR})
+  find_library(NCURSES_LIBRARY
+    NAMES ncurses
```

Comment:
Replace with `curses`. `ncurses` is open-source distribution of `curses`, `ncurses` must provide `curses` (or it's an error made by package author)